### PR TITLE
feat: add keyboard input UI and app theme

### PR DESCRIPTION
**Pull request by nayancprakash**
**Changes**
- Add UI for Chat keyboard input
- Add app_controller level theming
- Add chat timestamp to the last message
- Refactor some of chat

**UI**
> Simulator Screen Shot - iPhone 11 Pro - 2020-06-04 at 13 13 14

### CODE REVIEW

File: lib/navigation/app_controller.dart [hunk 1]
```diff
@@ -34,8 +34,12 @@ class _MyAppState extends State<AppController> {
...
           animationDuration: Duration(milliseconds: 200),
           animationCurve: Curves.bounceInOut
-        ), 
+        ),
       ),
+      theme: ThemeData(
+        primaryColor: Color(0xFF0F1236),
+        accentColor: Color(0xFF45cab9),
+      )
     );
```
Comment (eivenlour) [inline lib/navigation/app_controller.dart:43]: definitely a better way to store colors than making them global constants! thanks for this! btw, can you also remove line 3 that imports the color constants file

File: lib/chat/chat.dart [hunk 2]
```diff
@@ -65,7 +65,7 @@ buildChatView() {
       buildHeader(),
```
Comment (eivenlour) [inline lib/chat/chat.dart:65]: It's not letting me highlight the lines but in lines 36 and 47, you can probably place those string constants in a constant file, if youd like

File: lib/chat/user_message_rows.dart [hunk 3]
```diff
@@ -47,8 +47,8 @@ const dummyData = [
   },
 ];
 
-class Messages extends StatelessWidget {
-  Messages({Key key}) : super(key: key);
+class UserMessageRows extends StatelessWidget {
```
Comment (eivenlour) [inline lib/chat/user_message_rows.dart:50]: same here (its not letting me hightlight 😭 ), you can also place "Messages" in a constant file

**Comment by eivenlour** [comment]
I just ran your branch on my machine and I have a couple more minor change requests if you don't mind!

-  I noticed that the clicking target region in a message row is just the areas with text/photo. I think it's more user-intuitive if we make the whole row clickable

- Also super minor but can you add more padding/margin between the sent icon and the message 
Screen Shot 2020-06-04 at 2 48 39 PM

Other than that, I really think youve outdone yourself in this one bruv!!! I love the details esp the sound fx 👌

File: lib/navigation/app_controller.dart (hunk 1 above)
Comment (nayancprakash) [reply lib/navigation/app_controller.dart:43]: i deleted this line locally? some git issue i think isn't picking up the deleted line

**Pull request merged**